src/ai/backend/storage/server.py

```diff
@@ -89,6 +92,8 @@ async def server_main(
     manager_api_runner = web.AppRunner(manager_api_app)
     await client_api_runner.setup()
     await manager_api_runner.setup()
+    monitor_task = asyncio.create_task(monitor(ctx))
+    asyncio.ensure_future(monitor_task)
```

Comment:
`ensure_future()` is deprecated and it's recommended to use `asyncio.create_task()`.